bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/discovery/FreeboxOsDiscoveryService.java

```diff
@@ -181,7 +194,8 @@ private void discoverHosts(ThingUID bridgeUID, List<LanHost> lanHosts) {
                         bridgeUID, mac.toHexString(false));
                 logger.debug("Adding new Freebox Network Host {} to inbox", thingUID);
                 DiscoveryResultBuilder builder = DiscoveryResultBuilder.create(thingUID).withBridge(bridgeUID)
-                        .withLabel(lanHost.getPrimaryName().orElse("Network Device %s".formatted(macString)))
+                        .withLabel(lanHost.getPrimaryName()
+                                .orElse("@text/discovery.netwrok-device.label [ \"%s\" ]".formatted(macString)))
```

Comment:
```suggestion
                                .orElse("@text/discovery.network-device.label [ \"%s\" ]".formatted(macString)))
```

Typo, also need a fix in properties file.